WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'f4f6aa14-b5b6-4593-8259-b0801577bc35', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:45 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'dedf7307-68b3-474e-8379-305398d76353', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:46 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'd122d6ec-cdd4-4acb-b836-ec7aa4619f5b', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:47 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '4ded42b2-9b07-42f0-9d3d-b5f580b00741', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:48 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'e29b38ea-7362-42d9-8d57-4ffbe40af813', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:49 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '409e3acc-efda-4947-b283-cc8ef16637eb', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:50 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'd7095d46-2fd4-4df7-9588-9b0d58a49546', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:51 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '6aa52726-db26-4bf8-b0cd-a8201a8592d4', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:52 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '4225c86a-7766-4a10-8e55-7c2acd3ff302', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:53 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'f9bc0c76-7ee2-48b9-94e0-45be79770cff', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:54 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'b6ece3b2-3d0b-4e4a-be12-4e7f78b66900', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:55 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '6d3fb244-ddff-4abe-b195-27e47994d850', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:56 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '49b59ee1-c6db-433a-90f0-9487103dddbb', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:57 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '0e7b8169-6fb1-4abc-98e4-a01ba8b879af', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:58 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'bb2f0d43-b353-4c71-a808-41a115c19ed0', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:20:59 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '0b039d38-aa3c-4489-9bd4-21cd2142b0db', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:00 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'e4b38003-272c-403b-9887-d6508a81ced0', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:01 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '2281af22-b200-4bb3-9a6b-b53ac2ede04c', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:02 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '2ae3066f-38e3-434a-823c-05d6f000fac2', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:03 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'daed058c-cc6c-4dd6-bf6b-4cd104b666bd', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:04 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'e29fe2c6-d1e3-4c5a-8dea-df1ccc77b7bd', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:05 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '9114bd1a-1357-4547-b3a2-7274de38cffb', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:06 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '9484a6c1-2194-45a1-bc4c-2213c65b23d7', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:07 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '3cb4eb42-9609-4858-a21c-28548d952689', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:08 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '9658ebbd-84f7-4976-a7dc-7e09c1fa9bc0', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:09 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '1618d005-51e5-4b28-a758-f9153abdf88a', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:10 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '306d0090-b2fb-4663-a802-d12b865cfe8e', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:11 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '79880957-b43d-4c2a-82a1-d90aa4df29fc', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:12 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '14b788e5-f51b-47c2-92b8-c6f33cbaa07f', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:13 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '0d5abed2-6508-4852-a8fd-36fc987ba13b', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:14 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '82125db1-99e2-45e2-beda-5122a8d3da6d', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:15 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'f86c2b36-06d7-4cc1-86af-378fd9f486d4', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:16 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'd4d1ed45-30b1-4c86-a7c8-43b544ac7e44', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:17 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'd090d1cd-7f3b-49df-a74e-59941df778d5', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:18 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'bf1d3370-2a87-40a5-b54e-2d386aba6f48', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:19 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '2ddce0a8-88b8-4e7d-98d4-f6f38d839b7f', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:20 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '46c4718f-624f-47c3-b904-94813f95206f', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:21 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'e7980120-833b-4b37-bbe5-850776358866', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:22 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '5c4e2fe7-0526-412d-a0b5-2fd949a2eed1', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:23 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'e75604d0-6032-4d42-855b-886ed3067996', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:24 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '26c6073a-8a30-4f0c-b2e3-cb952cfdbc9f', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:25 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '1e89c26c-a3cd-467b-a6fb-beb984ad0a2e', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:26 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'bbdffbf4-b1be-49ac-a36e-2b5289499ade', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:27 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '2558cd45-4313-4efa-a5fd-905512056634', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:28 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'd928f5cd-f769-4db2-8b50-2c977244a556', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:29 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '53d784ab-5251-4e0d-8bf0-53fcadbebf57', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:30 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'c85ec6ca-5367-40b2-a503-280c52955d8a', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:31 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '586db852-fb56-4cd6-8e82-bb451bddfb5a', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:32 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '83f79eee-17ca-4e0f-bb62-d395274c6fe8', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:33 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '222eae8a-69b8-47a6-b007-1c27b6ea4e02', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:34 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '00f6b45d-caa8-44fa-b50c-e4e0401d47b1', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:35 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'd64fb4ca-714a-47ed-b2f3-d37bfb745fb2', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:36 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'c8f158c3-2d72-43c3-96a3-90582688512e', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:37 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'a80485bf-8c33-4c59-a536-b746060bf142', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:38 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '5a6135f9-e532-4dc8-822f-48d760e63c48', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:39 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'ba81737e-6361-44e6-adba-4ce00fe119ca', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:40 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '39043f8e-ad00-487d-be81-9251d8c6627d', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:41 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'b6366d76-e387-42d2-9e03-340861a6054e', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:42 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'b41a4d08-45af-4725-90ca-d44c0f70d45c', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:43 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}


WARNING  kubetest:utils.py:139 got api exception while waiting: (404)
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': 'a2eed8ac-3a68-4915-a6e7-ab253c90df7b', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': 'f721a1de-db86-41dd-85ff-024fa03f1932', 'X-Kubernetes-Pf-Prioritylevel-Uid': '306a3184-18b3-4dea-a773-bae71de7c462', 'Date': 'Sat, 21 Sep 2024 12:21:44 GMT', 'Content-Length': '204'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"namespaces \"context-cyan\" not found","reason":"NotFound","details":{"name":"context-cyan","kind":"namespaces"},"code":404}